rust/numaflow-core/src/config/monovertex.rs

```diff
@@ -94,6 +94,7 @@ impl MonovertexConfig {
             .as_ref()
             .and_then(|source| source.transformer.as_ref())
             .map(|_| TransformerConfig {
+                concurrency: batch_size as usize, // FIXME: introduce a new config called udf concurrency in the spec
```

Comment:
Do we need to?

Reply:
this is to make sure we can achieve X workers in parallel immaterial of the messages from previous vertex. we do read-ahead to achieve X concurrency/parallelism.